README.md

```diff
@@ -177,9 +186,58 @@ Objects<Person>().Delete(x => x.Age > 40);
 Objects<Person>().DeleteAll();
 ```
 
-## Roadmap
-
-Look at [Starcounter.Linq#6](https://github.com/Starcounter/Starcounter.Linq/issues/6)
+## Supported features
+
+- `SELECT` clause, also by using `IQueryable.Select` method
+- `WHERE` clause by using methods:
+  - `IQueryable.Where`
+  - `IQueryable.First`
+  - `IQueryable.FirstOrDefault`
+  - `IQueryable.Single`
+  - `IQueryable.SingleOrDefault`
+  - `IQueryable.Count`
+  - `IQueryable.Any`
+- `ORDER BY` clause by using methods:
+  - `IQueryable.OrderBy`
+  - `IQueryable.OrderByDescending`
+  - `IQueryable.ThenBy`
+  - `IQueryable.ThenByDescending`
+- Logical operators `AND`, `OR`, `NOT` by using `&&`, `||`, `!` operators in LINQ queries
+- Comparison by using:
+  - operators `=`, `>`, `>=`, `<`, `<=`, `<>`
+  - method `Object.Equals`
+- `LIKE` operator by using methods:
+  - `String.Contains`
+  - `String.StartsWith`
+  - `String.EndsWith`
+- `IEnumerable.Contains` method in LINQ queries
+- `DbHelper.GetObjectNo` method in LINQ queries
+- `IS` operator by using `is` operator in LINQ queries
+- `OFFSET` clause by using `IQueryable.Skip` method
+- `FETCH` clause by using methods:
+  - `IQueryable.Take`
+  - `IQueryable.FirstOrDefault`
+  - `IQueryable.First`
+  - `IQueryable.Single`
+  - `IQueryable.SingleOrDefault`
+- `COUNT` function by using method `IQueryable.Count`
+- `AVG` function by using method `IQueryable.Average`
+- `MIN` function by using method `IQueryable.Min`
+- `MAX` function by using method `IQueryable.Max`
+- `SUM` function by using method `IQueryable.Sum`
+- Calculating expression values in LINQ queries
+- Support First, FirstOrDefault, Single, SingleOrDefault and Any methods - ~#27~

```

Comment:
Should these words be marked as code with back-ticks as well?

Reply:
Why do you an issue tag at the end for this one?

Reply:
It was left by mistake, thanks for the catch. Removed it.